src/common/util.js

```diff
@@ -204,6 +204,19 @@ const util = {
     return obj
   },
 
+  set(src, path, value) {
```

Comment:
`set` is basically complimentary to `get` above. for now only used in plugins-ui, might be forgotten if left there. it's use is for supporting complex/nested config objects - not just simple key-value pairs of properties.

Reply:
Well, `get` is universally useful; `set` much less so. `get` can access properties of anything (arrays, strings, custom classes, ...) `set` is just for plain objects. Also `get` is just syntactic sugar for common access to optional props; whereas I would think that setting a deeply nested property is an error when the surrounding object does not exist in most cases. It's something that seems useful, but in my experience will be abused and is a good way to obfuscate the root cause of errors.